api/v1beta1/baremetalset_types.go

```diff
@@ -47,6 +52,70 @@ type BaremetalHostStatus struct {
 	Online                bool   `json:"online"`
 }
 
+// HardwareReqs defines request hardware attributes for the BaremetalHost replicas
+type HardwareReqs struct {
+	CPUReqs  CPUReqs  `json:"cpuReqs,omitempty"`
+	MemReqs  MemReqs  `json:"memReqs,omitempty"`
+	DiskReqs DiskReqs `json:"diskReqs,omitempty"`
+}
+
+// CPUReqs defines specific CPU hardware requests
+type CPUReqs struct {
+	// Arch is a scalar (string) because it wouldn't make sense to give it an "exact-match" option
+	Arch     string      `json:"arch,omitempty"`
+	CountReq CPUCountReq `json:"countReq,omitempty"`
+	MhzReq   CPUMhzReq   `json:"mhzReq,omitempty"`
```

Comment:
does the hw status reflect numa nodes? that'd be some good selection

Reply:
Unfortunately numa information is not gathered by the IPA, or at least is not placed in the BMH status